.travis.yml

```diff
@@ -13,6 +13,9 @@ install:
   - travis_retry pip install --upgrade pip
   - travis_retry pip install --upgrade tox tox-travis codecov
 
+before_script:
+  - python tools//ensure_codegen_up_to_date.py
```

Comment:
i'm surprised you need 2 forward slashes.

Reply:
The build failed previously when I used one backslash, so Shreyas suggested using forward slashes and escaping.

Reply:
Backslash fails because this does not run on a Windows system. Backslashes need escaping in some circumstances.

This is a forward slash.

Can you verify you need it twice?

Reply:
I can try a run with one forward slash